libs/common-services/src/test/java/uk/gov/di/ipv/core/library/helpers/JwtHelperTest.java

```diff
@@ -43,36 +36,30 @@
 
 @ExtendWith(MockitoExtension.class)
 class JwtHelperTest {
-    private final ObjectMapper objectMapper = new ObjectMapper();
+    private final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
+
+    @Data
+    private static final class ExamplePayload {
```

Comment:
(there's no reason for this test to use shared claims - it's just checking that we're using Nimbus correctly)